root/var/lib/vz/snippets/exec-cmds

```diff
@@ -69,6 +85,14 @@ exec_cmds() {
         if_action post-start exec_cpu_taskset "$ARG1"
         ;;
 
+      "#set_halt_poll")
```

Comment:
Can you add and document these options in `README`?

Reply:
Yes, I will document them and give an example.